src/main/webapp/app/account/activate/activate.component.ts

```diff
@@ -32,6 +31,6 @@ export class ActivateComponent implements OnInit {
     }
 
     login() {
-        this.modalRef = this.loginModalService.open();
+        this.router.navigate(['login']);
```

Comment:
Since now it is a proper page with its URL, why don't you just put the `href` in the `<a>` element instead of relying on JS? In this way this page has one less dependency

Reply:
good point 👍

Reply:
done